### PR TITLE
Update user doc to be in sync with code

### PR DESCRIPTION
**Pull request by hassanxp**
Added new section for initial setup, explaining env variables
and how to create an initial pfsDesign.
Revised makeSim example, with updated explanation of options
for that example.

### CODE REVIEW

File: simulatorUser/use.rst [hunk 1]
```diff
@@ -7,29 +7,30 @@ The main interface is a script to generate a single image.
...
 
-Single image
-------------
+``makePfsDesign``
+~~~~~~~~~~~~~~~~~
 
-The ``makeSim`` script runs the Simulator to produce a single image
-and the associated ``pfsConfig`` file.
-An example command-line is::
+The simulator requires a ``pfsDesign`` (or ``pfsConfig``)
```
Comment (PaulPrice) [inline simulatorUser/use.rst]: No, it will only accept a `pfsDesign` file.

File: simulatorUser/use.rst [hunk 2]
```diff
@@ -7,29 +7,30 @@ The main interface is a script to generate a single image.
...
-    makeSim --detector r1 --field lamObj --exptime 900 --expId 32 --imagetyp OBJECT --allOutput --pdb
+    makePfsDesign --fibers lam --pfsDesignId 1 --scienceCatId 1 --scienceObjId "18 55 71 76 93 94 105 112 115"
 
-The ``--detector`` specifies the spectrograph and arm;
-we currently only support ``r1``.
+The above command generates a file ``pfsDesign-0x0000000000000001.fits``, corresponding to a
+``pfsDesignId`` of ``1``, 
+mapping randomly objects with object identifiers listed in the value for the 
+``--scienceObjId`` option. These object identifiers are associated with a catalogue '``1``' 
+(the value of ``--scienceCatId``).
```
Comment (PaulPrice) [inline simulatorUser/use.rst]: The simulator will then look under the directory `1` (the `scienceCatId`) for the input spectra.

File: simulatorUser/use.rst [hunk 3]
```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
...
-* ``Comb``: a set of emission lines of equal spacing for all fibers.
-* ``Const``: a constant :math:`F_nu` spectrum for all fibers.
-* ``Sky``: a pure sky spectrum for all fibers.
-* ``Obj``: a combination of sky, flux calibration sources and objects.
+Also before running the simulator the envionment variable ``DRP_INSTDATA_DIR`` needs to be set to point
+to a recent version of the ``drp_instdata`` git repository. For example::
+
+    cd /path/to
+    git clone https://github.com/Subaru-PFS/drp_instdata.git
+    export DRP_INSTDATA_DIR=/path/to/drp_instdata
```
Comment (PaulPrice) [inline simulatorUser/use.rst]: Better to `setup -jr /path/to/drp_instdata`.
Be warned that drp_instdata is a large package.

File: simulatorUser/use.rst [hunk 4]
```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
...
+The main simulator command, ``makeSim``, requires the corresponding object spectra
+as input. More specifically, the location of those input spectra.
+While this can be specified through the ``--spectraDir`` option, an alternative
+is to use the default value (which is the current working directory)
+and could create a symbolic link to your spectral files. 
+For example, if your input data are the ``lowz_COSMOS`` data 
+from the ``drp_instdata`` repository, you could create a link as follows::
+
+    ln -s /path/to/drp_instdata/data/objects/lowz_COSMOS/ 1
+where the ``1`` corresponds to the catalogue identifer of the source (eg the value of
```
Comment (PaulPrice) [inline simulatorUser/use.rst]: "eg" --> "i.e."

File: simulatorUser/use.rst [hunk 5]
```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
...
 This is used in the header.
 
-``--allOutput`` specifies that the Simulator
-should write additional FITS extensions
-containing information.
-
 ``--pdb`` specifies that the program should drop into the debugger in the event of an exception.
 
-There are additional arguments,
-but I don't know what's working and what's not.
```
Comment (PaulPrice) [inline simulatorUser/use.rst:68]: This is probably still true....

File: simulatorUser/use.rst [hunk 6]
```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
...
+
 ``--expId`` specifies the exposure identifier to use.
 This is used in the filename.
 
 ``--imagetyp`` specifies the image type.
 This is used in the header.
 
-``--allOutput`` specifies that the Simulator
-should write additional FITS extensions
-containing information.
```
Comment (PaulPrice) [inline simulatorUser/use.rst:63]: Suggest leaving this in.

File: simulatorUser/use.rst [hunk 7]
```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
...
+
+
+Single image
+------------
+
+The ``makeSim`` script runs the Simulator to produce a single image
+and the associated ``pfsConfig`` file.
+An example command-line is::
+
+    makeSim --detector r1 --pfsConfig --pfsDesignId 1 --exptime 0 --domeClosed --expId 0 --imagetyp BIAS
```
Comment (PaulPrice) [inline simulatorUser/use.rst:83]: `--pfsConfig` isn't explained below.
I think `--expId` has been replaced with `--visit`.

File: simulatorUser/use.rst [hunk 8]
```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
...
+``pfsDesign-0xNNNNNNNNNNNNNNNN.fits`` file present in the current working directory,
+where ``NNNNNNNNNNNNNNNN`` is the 16-digit hexadecimal representation of the 
+designId you have
+specified in the ``--pfsDesignId`` argument.
 
 ``--exptime`` specifies the exposure time in seconds.
 This is used to scale the input spectrum.
 
+``--domeClosed`` determines whether the sky background is included or not. 
+If ``True``, the sky background will not be included.
```
Comment (PaulPrice) [inline simulatorUser/use.rst]: This is a command-line description, so `True` has no meaning.

File: simulatorUser/use.rst [hunk 9]
```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
...
+------------
+
+The ``makeSim`` script runs the Simulator to produce a single image
+and the associated ``pfsConfig`` file.
+An example command-line is::
+
+    makeSim --detector r1 --pfsConfig --pfsDesignId 1 --exptime 0 --domeClosed --expId 0 --imagetyp BIAS
+ 
+The ``--detector`` specifies the spectrograph and arm;
+we currently only support ``r1``.
```
Comment (PaulPrice) [inline simulatorUser/use.rst]: And `b1`.

**Review by PaulPrice — APPROVED** [review]
A few suggestions.
Usually I would suggest separating the different styles of edits (cleanup vs adding information), but this is your package, so I won't insist... (:

File: simulatorUser/use.rst (hunk 1 above)
Comment (hassanxp) [reply simulatorUser/use.rst]: Fixed.

File: simulatorUser/use.rst (hunk 2 above)
Comment (hassanxp) [reply simulatorUser/use.rst]: Fixed.

File: simulatorUser/use.rst (hunk 3 above)
Comment (hassanxp) [reply simulatorUser/use.rst]: Good point. Fixed.

File: simulatorUser/use.rst (hunk 4 above)
Comment (hassanxp) [reply simulatorUser/use.rst]: Fixed.

File: simulatorUser/use.rst (hunk 5 above)
Comment (hassanxp) [reply simulatorUser/use.rst:68]: Ok, re-introduced but re-worded.

File: simulatorUser/use.rst (hunk 6 above)
Comment (hassanxp) [reply simulatorUser/use.rst:63]: Ok, re-introduced.

File: simulatorUser/use.rst (hunk 7 above)
Comment (hassanxp) [reply simulatorUser/use.rst:83]: I've explained ``--pfsConfig`` now. 
I've not switched from ``--expId`` to ``--visit`` as your changes are not currently merged to master.

File: simulatorUser/use.rst (hunk 8 above)
Comment (hassanxp) [reply simulatorUser/use.rst]: Fixed.

File: simulatorUser/use.rst (hunk 9 above)
Comment (hassanxp) [reply simulatorUser/use.rst]: Fixed.

**Pull request merged**